mitodl/bootcamp-ecommerce
# remove run key from line

#### What are the relevant tickets?
closes https://github.com/mitodl/bootcamp-ecommerce/issues/984
In https://github.com/mitodl/bootcamp-ecommerce/pull/971 we added `bootcamp_run_id` to the `Line` model to be used as a foreign key. We can now remove `run_key` from `Line` so there isn't a duplicate foreign key

#### What's this PR do?
This pr removes `run_key` from `ecommerce_line`

#### How should this be manually tested?
Verify that the migration is reversible
Verify that you can create charges and refunds
`ecommerce.api.serialize_user_bootcamp_run()` should work as expected
`LineSerializer(Line.objects.last()).data` should return the run_key as expected
`http://bc.odl.local:8099/applications/<aplication id>/payment-history/` should be unchanged
`HubspotDealSerializer(BootcampApplication.objects.last()).data` should return the expected data
